### PR TITLE
PRO-1861 - Error handling and apollo/client package fixes

### PR DESCRIPTION
**Pull request by vignesha22**
## Description

- Added possibleSolution parameter for the errors that are commonly returned from the bundler which are taken from the eip4337 error codes
- Updated package dependencies for the apollo/client
- Updated config.yml to point to this branch for running e2e tests temporarily.

## How Has This Been Tested?

- Tested Locally and build also was successful

## Screenshots (if appropriate):
Screenshot 2023-10-31 at 5 45 37 AM
Screenshot 2023-10-31 at 5 42 41 AM

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### CODE REVIEW

**Comment by ch4r10t33r** [comment]
@vignesha22 pls can you address the issue related to failing e2e tests?

**Review by ch4r10t33r — APPROVED** [review]
LGTM

File: src/sdk/errorHandler/constants.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+export const errorMsg = {
+    '-32521': 'Check for balance in your Smart wallet', // execution reverted
+    '-32500': `Please make sure you have enough funds for wallet creation.`, // transaction rejected by entryPoint's simulateValidation, during wallet creation or validation
+    '-32501': `Check paymaster data`,  // transaction rejected by paymaster's validatePaymasterUserOp
+    '-32502': 'Try with another bundler', //transaction rejected because of opcode validation
+    '-32503': 'Try with valid validUntil and validAfter values', // UserOperation out of time-range
```
Comment (ch4r10t33r) [inline src/sdk/errorHandler/constants.ts:6]: validUntil and validAfter cannot be past timestamps.

**Comment by vignesha22** [comment]
Please ignore the package version update as 1.3.9 is not yet published as that merge with master failed to build because of dependencies outdated @0xSulpiride @lbw33 @kaushalrajbacancy

**Review by kaushalrajbacancy — APPROVED** [review]

File: src/sdk/api/api.service.ts [hunk 2]
```diff
@@ -154,7 +145,7 @@ export class ApiService extends Service {
 
       try {
         // checks and removes outdated cache entries
-        this.cache.expire();
+        // this.cache.expire();
```
Comment (lbw33) [inline src/sdk/api/api.service.ts:148]: Do we still need this line?

**Review by lbw33 — APPROVED** [review]

File: src/sdk/api/api.service.ts (hunk 2 above)
Comment (vignesha22) [reply src/sdk/api/api.service.ts:148]: nope since the cache is managed in memory rather than handling by a separate policy package

Comment (vignesha22) [reply src/sdk/api/api.service.ts:148]: will remove that comment line

**Pull request merged**